website/routes.py

```diff
@@ -858,6 +855,30 @@ def make_url_map(app):
             project_views.node.node_registrations,
             OsfWebRenderer('project/registrations.mako', trust=False)
         ),
+        Rule(
+            [
+                '/project/<pid>/registrations/',
+                '/project/<pid>/node/<nid>/registrations/',
+            ],
+            'post',
+            project_views.drafts.new_draft_registration,
+            OsfWebRenderer('project/draft_registration.mako', trust=False)),
+        Rule(
+            [
+                '/project/<pid>/drafts/<draft_id>/',
+                '/project/<pid>/node/<nid>/drafts/<draft_id>/',
+            ],
+            'get',
+            project_views.drafts.edit_draft_registration_page,
+            OsfWebRenderer('project/edit_draft_registration.mako', trust=False)),
+        Rule(
+            [
+                '/project/<pid>/drafts/<draft_id>/register/',
+                '/project/<pid>/node/<nid>/drafts/<draft_id>/register/',
+            ],
+            'get',
+            project_views.drafts.draft_before_register_page,
+            OsfWebRenderer('project/register_draft.mako', trust=False)),
```

Comment:
They seem fine as routes on their own. If we didn't have to add routes for web renderer things, it would be better. 

A post route for a non-api item seems fairly old-fashioned to me and sounds like it's going to be difficult to convert that to the v2 api in a sprint or two. I would also kind of expect it to be a post rout to /drafts/ if you're creating a new draft. 

The edit drafts page is fine if we're not just handling that with knockout or similar. Again: old fashioned compared with modern OSF ways, so I'm wondering if that will get changed as part of other aspects of the code review. If that's the best way, then that route is okay, but it'll be something that has to be maintained.  Same with the register route, really.

So, in summary: if we need them, the routes are fine, but it seems weird to me that we need them.